src/tests/java/picard/sam/MergeBamAlignmentTest.java

```diff
@@ -106,8 +105,8 @@ public void testMergerWithSupplemental() throws Exception {
                 false, true, false, 1,
                 "0", "1.0", "align!", "myAligner",
                 true, fasta, outputWithSupplemental,
-                SamPairUtil.PairOrientation.FR, null, null, null
-        );
+                SamPairUtil.PairOrientation.FR, null, null, null,
```

Comment:
instead of changing so many lines, coudl you add another version of doMergeAlignment that takes one argument less?